sphinx_gallery/gen_rst.py

```diff
@@ -737,23 +782,11 @@ def __call__(self):
 
 
 def _get_memory_base():
-    """Get the base amount of memory used by running a Python process."""
+    """Get the base amount of memory used by the current Python process."""
     # There might be a cleaner way to do this at some point
```

Comment:
Probably can do away with this comment now that this is a 2 liner function :sweat_smile: